app/views/users/index.html.erb

```diff
@@ -1,32 +1,22 @@
-  
 <section class="users-section">
   <ul class="users-list">
     <% @users.each do |user| %>
-      <% if user != current_user %>
       <li>
-         Name: <%= user.name %>
+        Name: <%= user.name %>
         <span class="profile-link">
-          <%= link_to 'See Profile',  user_path(user), class: "btn btn-secondary" %>
-           <% if current_user.friend_invites(user) %>
-            <button class="btn btn-warning"> Pending Invitation </button>
-          <% elsif current_user.receive_invitation(user)%>
-            <%= link_to 'Accept', accept_invite_path(user_id: user.id), class:"btn btn-primary" %>
-            <%= link_to 'Decline', reject_invite_path(user_id: user.id), method: :delete, class:"btn btn-danger" %>
-          <% elsif !current_user.friend?(user) && current_user.id != user.id %>
-            <%= link_to 'Add Friend', send_invite_path(user_id: user.id) , class:"btn btn-success" %>
-            <% elsif current_user.friend?(user) %>
-            <%= link_to 'Remove friend', remove_friend_path(user_id: user.id), method: :delete, data: {confirm: "Are you sure you want to remove friend?"}, class: "btn btn-danger" %>
-          <% end %>
+          <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %> | 
         </span>
+        <% if current_user != user && friends_check(user) == false && pending_request(user) == false %>
+          <%= link_to 'Send friend request', "/users/#{user.id}/send_friend", class: 'profile-link' %>
+        <% elsif pending_request(user) %>
+          <span class= 'profile-link'>Friend request pending</span> 
+          <% if !@pending_request.blank? %>
+            <span class= 'profile-link'><%= accept_friend(@pending_request) %> | <%= reject_friend(@pending_request) %></span>
+          <% end %>
+        <% elsif friends_check(user) %>
+          <span class= 'profile-link'>Friends!!!</span> <span class= 'profile-link'><%= unfriend(user) %></span>
+        <% end %>
```

Comment:
- [ ] use a helper method to put the if/else statement and render the links within the helper instead of having the logic in the view